CONTRIBUTING.md

```diff
@@ -1,83 +1,8 @@
 # How to contribute
 
-Contributions from the community are highly encouraged.
+We encourage contributions from the community.
+When creating content, please follow our [style guidance](/STYLE.md).
 
 To ensure that each change is relevant and properly peer reviewed, we request that you adhere to best practices for open-source contributions.
-This means that if you are outside the Temporal organization, you must fork the repository and create pull requests from branches on your own fork.
-GitHub's [first-contributions repo README](https://github.com/firstcontributions/first-contributions) provides an example of how to do that.
-
-When it comes to crafting content, please follow our [style guidance](#style-guidance).
-
-## Preview changes locally
```

Comment:
I think we should still keep the "How to install and run this site locally" info in the README or CONTRIBUTING - but within context

Reply:
Right-o.